test/jdk/ProblemList.txt

```diff
@@ -787,7 +786,7 @@ sun/util/locale/provider/CalendarDataRegression.java            8268379 macosx-x
 
 # svc_tools
 
-sun/tools/jhsdb/BasicLauncherTest.java                          8211767 linux-ppc64,linux-ppc64le
+sun/tools/jhsdb/BasicLauncherTest.java                          8228649 linux-ppc64,linux-ppc64le
```

Comment:
8228649 is also resolved. Maybe this isn't an issue anymore. Would need to ask someone on the PPC team.

Reply:
Yes, there were a couple of them that was duplicates of closed bugs, but this change only deals with duplicates. To actually remove the tests from the list I would need to run them for a while to make sure they pass now. That's a later task.